zk_toolbox/crates/zk_supervisor/src/commands/lint.rs

```diff
@@ -19,6 +19,8 @@ pub struct LintArgs {
     pub check: bool,
     #[clap(long, short = 'e')]
     pub extensions: Vec<Extension>,
+    #[clap(long, default_missing_value = "true", num_args = 0..=1)]
+    pub contracts: Option<bool>,
```

Comment:
Why do you need it? why not `sol`?

Reply:
In the previous tool it was separated between `sol` files in the era repo and files in the contracts repo. The contract option lints files in the contract repo

Reply:
Them probably we need to rename extensions, and add it there?

Reply:
Makes sense!